### DISCUSSION TITLE
Impersonating a user terminates session

### DISCUSSION THREAD

**Discussion by Brammm** [symfony/symfony#45518] [Q&A]
I've built in switch user functionality in my app. Locally this works fine. When deployed to Google Cloud (which does use Cloud Armor), it no longer works. After clicking the impersonate button, the active user is logged out and you're redirected to the login page. I'm not getting an exception or anything...

I'm a little stuck debugging this, anybody could point me in the right direction of where to look or what might be causing this?

**Comment by lyrixx** [comment]
You could check the cookie header value, to ensure nothing wrong occur here.
You can also enable all log in symfony to understand what is going on.
